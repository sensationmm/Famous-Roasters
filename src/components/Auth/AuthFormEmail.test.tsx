import { MockedProvider } from '@apollo/client/testing'
import { render, waitFor } from '@testing-library/react'
import React from 'react'

import { AuthFormEmail } from '.'

describe('Email fields group custom auth component', () => {
  const snippet = () => (
    <form name="testAuthFormItemInput">
      <AuthFormEmail screenKey="signIn" onChange={() => null} />
    </form>
  )

  it('Renders correctly', async () => {
    const { container } = render(
      <MockedProvider defaultOptions={{ watchQuery: { fetchPolicy: 'no-cache' } }}>{snippet()}</MockedProvider>,
    )
    await waitFor(() => new Promise((res) => setTimeout(res, 0)))
    expect(container).toMatchSnapshot()
  })
})
