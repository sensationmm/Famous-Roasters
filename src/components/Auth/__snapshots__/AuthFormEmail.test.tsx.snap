// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Email fields group custom auth component Renders correctly 1`] = `
<div>
  <form
    method="POST"
  >
    <label
      class="relative"
    >
      <span
        class="absolute -top-6 left-3 px-1 bg-white"
      >
        <span
          class="text-sm leading-5"
        >
          Deine E-mail-Adresse
        </span>
      </span>
      <input
        class="w-full border rounded-full px-4 py-2 focus:outline-0 focus:ring focus:ring-primary/30"
        data-testid="email"
        placeholder="Deine E-mail-Adresse"
        type="email"
        value=""
      />
    </label>
  </form>
</div>
`;
